src/lib/markbind/src/parser.js

```diff
@@ -22,15 +22,13 @@ const utils = require('./utils');
 cheerio.prototype.options.xmlMode = true; // Enable xml mode for self-closing tag
 cheerio.prototype.options.decodeEntities = false; // Don't escape HTML entities
 
-const ATTRIB_INCLUDE_PATH = 'include-path';
-const ATTRIB_CWF = 'cwf';
+const {
+  ATTRIB_INCLUDE_PATH,
+  ATTRIB_CWF,
+  BOILERPLATE_FOLDER_NAME,
+  IMPORTED_VARIABLE_PREFIX,
+} = require('../../../constants');
```

Comment:
`src/lib/markbind/*/*` (MarkBind core library) should not require from `../../../*` (MarkBind CLI).

Let's create `src/lib/markbind/src/constants.js` too.